### PR TITLE
autoload.lua ignores all files starting with '.'

### PR DESCRIPTION
**Pull request by yuezhu**

### CODE REVIEW

**Comment by yuezhu** [comment]
OS X system automatically creates those hidden files. It doesn't make sense to add them into playlist.

**Comment by haasn** [comment]
It would probably be a justified idea to ignore files starting with `.` in general.

File: TOOLS/lua/autoload.lua [hunk 1]
```diff
@@ -72,6 +72,9 @@ function find_and_add_entries()
         return
     end
     table.filter(files, function (v, k)
+        if string.match(v, "^%._.*") then
```
Comment (wiiaboo) [inline TOOLS/lua/autoload.lua]: Change this to ignore all files starting with "." as suggested.

**Comment by yuezhu** [comment]
It makes sense to ignore all hidden files starting with `.` .

**Comment by wiiaboo** [comment]
Great, now if possible, rebase to one single commit and use a more appropriate commit message, see the [git history of autoload.lua](https://github.com/mpv-player/mpv/commits/master/TOOLS/lua/autoload.lua) as example. Basically `TOOLS\autoload.lua: <summary of change>`.

**Comment by wm4** [comment]
This was apparently done, so merged it. (Saw it because I did "git fetch origin".)

**Pull request closed without merging**